.github/fixtures/test-fixtures-locally.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+
+if [ "$1" = "" ]; then
```

Comment:
It would be better to check variable existence instead of value. e.g. Can you replace it with `if [ -z "$1" ]; then`?

Reply:
Sure, but just out of curiosity, could you tell me the guidelines you follow?
I'm not familiar with the guidelines of shell scripts.

Reply:
e2a6a06f6db21b7eb4b6e7854f7ddb229b879178

Reply:
> Sure, but just out of curiosity, could you tell me the guidelines you follow? I'm not familiar with the guidelines of shell scripts.

I'm not really following guidelines here, just "best practices" that I see from here and there. (here and there == arch linux projects 😄 ) One thing that I can say is I use [shellcheck](https://github.com/koalaman/shellcheck) as a linter.